bundle-workflow/src/test_workflow/integ_test/local_test_cluster.py

```diff
@@ -92,7 +92,7 @@ def wait_for_service(self):
                 logging.info(f"Pinging {url} attempt {attempt}")
                 response = requests.get(url, verify=False, auth=("admin", "admin"))
                 logging.info(f"{response.status_code}: {response.text}")
-                if response.status_code == 200 and '"status":"green"' in response.text:
+                if response.status_code == 200:
                     logging.info("Cluster is green")
```

Comment:
Please fix this comment and other places within this code where "green" is used.

Reply:
That's right should fix this. Any suggestion you got for the comment?

Reply:
Cluster is green --> Service is available

Reply:
May be "Cluster is healthy"/"Cluster is ready for the test".

Reply:
Changed the message to `Service is available`